app/formatters/duty_expression_formatter.rb

```diff
@@ -19,10 +19,14 @@ def format(opts = {})
       measurement_unit_qualifier = opts[:measurement_unit_qualifier]
       measurement_unit_abbreviation = measurement_unit.try :abbreviation,
                                                            measurement_unit_qualifier: measurement_unit_qualifier
+      resolved_meursing_component = opts[:resolved_meursing]
```

Comment:
passing thought - would it make more sense to use named params instead of an opts hash? More control over which (if any) are required then

Reply:
I agree. I actually think we pull most of these things from the component and could do with passing just the component plus some options (like formatted and resolved_meursing). We'll want to ask the component what it's duty expression is and under which category of duty expression it sits in future for example.

DutyExpression#compound?